docs/core/tools/project-json-to-csproj.md

```diff
@@ -0,0 +1,592 @@
+> For more information, see the [dependency type](#dependency-type) section.
+
+### Top-level dependencies
+```json
+{
+  "dependencies": {
+    "Microsoft.AspNetCore": "1.1.0"
+  }
+}
+```
+
+```xml
+<ItemGroup>
+  <PackageReference Include="Microsoft.AspNetCore" Version="1.1.0" />
+</ItemGroup>
+```
+
+### Per-framework dependencies
+```json
+{
+  "framework": {
+    "net451": {
+      "dependencies": {
+        "System.Collections.Immutable": "1.3.1"
+      }
+    },
+    "netstandard1.5": {
+      "dependencies": {
+        "Newtonsoft.Json": "9.0.1"
+      }
+    }
+  }
+}
+```
+
+```xml
+<ItemGroup Condition="'$(TargetFramework)'=='net451'">
+  <PackageReference Include="System.Collections.Immutable" Version="1.3.1" />
+</ItemGroup>
+
+<ItemGroup Condition="'$(TargetFramework)'=='netstandard1.5'">
+  <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
+</ItemGroup>
+```
+
+### imports
+
+```json
+{
+  "dependencies": {
+    "YamlDotNet": "4.0.1-pre309"
+  },
+  "frameworks": {
+    "netcoreapp1.0": {
+      "imports": [
+        "dnxcore50",
+        "dotnet"
+      ]
+    }
+  }
+}
+```
+
+```xml
+<PropertyGroup>
+  <PackageTargetFallback>dnxcore50;dotnet</PackageTargetFallback>
+</PropertyGroup>
+<ItemGroup>
+  <PackageReference Include="YamlDotNet" Version="4.0.1-pre309" />
+</ItemGroup>
+```
+
+### dependency type
+
+#### type: project
+```json
+{
+  "dependencies": {
+    "MyOtherProject": "1.0.0-*",
+    "AnotherProject": {
+      "type": "project"
+    }
+  }
+}
+```
+
+```xml
+<ItemGroup>
+  <ProjectReference Include="..\MyOtherProject\MyOtherProject.csproj" />
+  <ProjectReference Include="..\AnotherProject\AnotherProject.csproj" />
+</ItemGroup>
+```
+
+> [!NOTE]
+> This will break the way that `dotnet pack --version-suffix $suffix` determines the 
+dependency version of a project reference.
+
+
+#### type: build
+```json
+{
+  "dependencies": {
+    "Microsoft.EntityFrameworkCore.Design": {
+      "version": "1.1.0",
+      "type": "build"
+    }
+  }
+}
+```
+
+```xml
+<ItemGroup>
+  <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="1.1.0" PrivateAssets="All" />
+</ItemGroup>
+```
+
+#### type: platform
+```json
+{
+  "dependencies": {
+    "Microsoft.NETCore.App": {
+      "version": "1.1.0",
+      "type": "platform"
+    }
+  }
+}
+```
+
+There is no equivalent in csproj. The `Microsoft.NETCore.App` metapackage is implicitly referenced based on the target framework of your app.
+For more information, see [Implicit package references](../preview3/tools/csproj.md#implicit-package-references).
+
+## runtimes
+```json
+{
+  "runtimes": {
+    "win7-x64": {},
+    "osx.10.11-x64": {},
+    "ubuntu.16.04-x64": {}
+  }
+}
+```
+
+```xml
+<PropertyGroup>
+  <RuntimeIdentifiers>win7-x64;osx.10-11-x64;ubuntu.16.04-x64</RuntimeIdentifiers>
+</PropertyGroup>
+```
+
+### Standalone apps (self-contained deployment)
+In project.json, defining a `runtimes` section mean the app was standalone during
```

Comment:
@natemcmaster I will as soon as I finish working on all your feedback. I'll let you know.